js/viz/docs/docStatic.js

```diff
@@ -136,6 +145,19 @@ var staticMethods = {
     * @export getPalette
     */
     getPalette: function() { },
+    /**
+    * @name vizmethods.generateColors
+    * @publicName generateColors(palette, options)
```

Comment:
The method should have three arguments: `palette`, `count`, and `options`; `options` should contain two fields: `paletteExtensionMode` of type `Enums.VizPaletteExtensionMode` and `baseColorSet` of a new type that you should declare in the `docEnums.js` file. I suggest that you call it `PaletteColorSet`